test/prides_test.dart

```diff
@@ -194,6 +194,20 @@ void main() {
     expect(find.byKey(backgroundKey), findsOneWidget);
   });
 
+  testWidgets(
+      'Display the slide number when set to true',
+      (tester) async {
+    // const backgroundKey = Key('backgroundWidget');
```

Comment:
Can remove this commented line. This alone won't solve your formatting. Please check the link I sent you in before comment.

```suggestion
```